src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -77,14 +85,20 @@ private static Command prepareTemplate(String[] args) throws DukeException {
             throw new DukeException("Invalid argument");
         }
 
-        Template template = getTemplate(args[1]);
+        template = getTemplate(args[1]);
         return new TemplateCommand(template);
     }
 
     private static Template getTemplate(String arg) throws DukeException {
         switch (arg) {
         case "rl":
             return new LrTemplate();
+        case "rc":
+            return new RcTemplate();
+        case "r":
+            return new RTemplate();
+        case "lc":
+            return new LcTemplate();
```

Comment:
While I know having break statements out here is kind of useless but does this violate coding standard? https://se-education.org/guides/conventions/java/basic.html#layout

Reply:
Nope! In fact, it cannot compile if you add the `break` statements.

Reply:
Oh wow that's...interesting